x-pack/plugins/lens/public/drag_drop/drag_drop.tsx

```diff
@@ -93,6 +93,11 @@ interface BaseProps {
    * Order for keyboard dragging. This takes an array of numbers which will be used to order hierarchically
    */
   order: number[];
+
+  /**
+   * don't display ghost image for the drop element
+   */
+  noGhost?: boolean;
```

Comment:
I don't have a strong feeling here, but we could call the prop `ghost` and treat `undefined` as true:
```
{(ghost || typeof ghost === 'undefined') &&
      dragging?.ghost &&
      activeDropTarg...
```